core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/run_sh.go

```diff
@@ -266,6 +266,10 @@ func (builtin *RunShCapabilities) FillPersistableAttributes(builder *enclave_pla
 	builder.SetType(RunShBuiltinName)
 }
 
+func (builtin *RunShCapabilities) Description() string {
+	return "Running a one time bash script"
```

Comment:
Could we make this `Running sh script` in the case where there's a lot of command to run, and `Running sh script: .....` if not?